downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -262,7 +262,9 @@ abstract class VersionControlSystem {
             }
         }
 
-        addGuessedRevision(pkg.id.name, pkg.id.version)
+        if (!addGuessedRevision(pkg.id.name, pkg.id.version) && pkg.id.type == "NPM" && pkg.id.namespace.isNotEmpty()) {
```

Comment:
Commit message:
- s/git tags/Git tags/ (first paragraph)
- The formatting for `[2]` looks a bit odd, how about just comma-separating the tag name in a single line?

Reply:
Please also add a short code comment that this is a fallback for Lerna workspaces.